CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change history for ui-data-import
 
-## [8.0.2] (IN PROGRESS)
+## [8.1.0] (IN PROGRESS)
```

Comment:
(UIDATIMP-1675) should be still under 8.0.2, it's gonna be released as ramsons bf